demo

```diff
@@ -96,6 +96,7 @@ export INTEGRATION_VERSION=$(git describe --tags --abbrev=0)
 export MENDER_ARTIFACT_VERSION=$(extra/release_tool.py -g mender-artifact)
 export MENDER_VERSION=$(extra/release_tool.py -g mender)
 export MENDER_DEB_PACKAGE_VERSION=$MENDER_VERSION
+export GATEWAY_IP=$(ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p')
```

Comment:
We should use `ip addr` here, not `ifconfig`. `ifconfig` is deprecated, and no longer present in RHEL7, nor in minimal Debian installs.

Also, use `sed -r`, not `sed -E`. The latter is BSD-only, and although GNU introduced `-r`, I believe it's compatible with both (at least FreeBSD's man page claims it is).

Reply:
unfortunately `ip` is not available on mac same as `sed -r` so now that had to get a bit messier 🙁